io-engine/src/grpc/v1/pool.rs

```diff
@@ -39,6 +39,28 @@ pub enum PoolProbe {
     UuidOrName(String),
     NameUuid { name: String, uuid: Option<String> },
 }
+impl PoolProbe {
+    fn name_uuid(name: &str, uuid: &Option<String>) -> Self {
+        Self::NameUuid {
+            name: name.to_owned(),
+            uuid: uuid.to_owned(),
+        }
+    }
+}
+pub(crate) enum Probe {
+    Kind(PoolBackend),
+    Probe(PoolProbe),
+}
```

Comment:
let's keep enum name and the enum variant name different. Perhaps call `enum ProbeType`

Reply:
Yes that's a better name :)